examples/video-chat/android/app/build.gradle

```diff
@@ -101,14 +101,18 @@ android {
     }
 }
 
+react {
+    hermesCommand = "../../node_modules/react-native/sdks/hermesc/%OS-BIN%/hermesc"
+}
+
 dependencies {
     // The version of react-native is set by the React Native Gradle Plugin
     implementation("com.facebook.react:react-android")
     implementation("com.facebook.react:flipper-integration")
 
 
     implementation("androidx.swiperefreshlayout:swiperefreshlayout:1.0.0")
-    implementation 'androidx.core:core-ktx:+'
```

Comment:
note: this might be affecting current android builds on `main` too, there was new release of `core-ktx` that is not compatible with our configuration.